pennylane/measurements.py

```diff
@@ -1235,6 +1268,63 @@ def circuit():
     return ShadowMeasurementProcess(Shadow, wires=wires, seed=seed)
 
 
+def shadow_expval(obs, k=1, seed_recipes=True):
+    r"""Compute expectation values using classical shadows in a differentiable manner.
+
+    The canonical way of computing expectation values is to simply average the expectation values for each local snapshot,
+    :math:`\langle O \rangle = \sum_t \text{tr}(\rho^{(t)}O) / T`. This corresponds to the case ``k=1``.
+    However, it is often desirable for better accuracy to split the ``T`` measurements into ``k`` equal parts to compute
+    the median of means, see `arXiv:2002.08953 <https://arxiv.org/abs/2002.08953>`_.
+
+    Args:
+        H (:class:`~.pennylane.Hamiltonian` or :class:`~.pennylane.operation.Tensor`): Observable to compute the expectation value over.
```

Comment:
We can't use `obs`, since that's being used internally by the tape to be replaced by a set of other qwc observables (actually I'm not entirely sure of the logic behind everything here.) Since the shadow expectation values are all being calculated classically, we don't need any of that.